README.rst

```diff
@@ -43,6 +43,7 @@ What just happened:
 
 - ``west update`` clones the other repositories named in the manifest file,
   creating working trees in the installation directory ``zephyrproject``.
+  It also checkouts submodules if cloned projects have defined any.
```

Comment:
It's OK to leave this out of the PR. I will handle writing the documentation (which lives in the zephyr repository) once it's merged, before the next west release. It's not quite true as written, because submodules aren't always checked out. It's opt-in.

Reply:
Ok, done.